packages/react/src/hooks/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './penumbra';
+export * from './service';
+export * from './transport';
```

Comment:
suggestion: use defined exports, e.g. `export { usePenumbra } from './penumbra';`. 

I couldn't find the link as a proof but apparently Vite's HMR lags for the star-exports and performs better with defined exports. Yet, no proof from my side, so feel free to discard it 😁